sources/packages/backend/apps/api/src/route-controllers/education-program/education-program.controller.service.ts

```diff
@@ -229,4 +226,43 @@ export class EducationProgramControllerService {
       hasOfferings,
     };
   }
+
+  /**
+   * Allows a program to be deactivated.
+   * @param programId program to be updated.
+   * @param auditUserId user that should be considered the one that is causing the changes.
+   * @param isActive value to update the isActive state for the program.
+   * @param options method options.
+   * - `institutionId` institution used for authorization.
+   * - `notes` notes associated with the change.
+   */
+  async deactivateProgram(
```

Comment:
Should we create an overload to be specific about what are accepted options that can be sent to the method? 
![image](https://github.com/bcgov/SIMS/assets/54600590/04ecd081-0b40-48c9-9de7-afd62ebe0e34)

Reply:
Added overloads.